app/controllers/jobseekers/saved_jobs_controller.rb

```diff
@@ -16,7 +16,8 @@ def destroy
   end
 
   def index
-    @saved_jobs = current_jobseeker.saved_jobs
+    @sort = SavedJobSort.new.update(column: sort_column, order: sort_order)
+    @saved_jobs = current_jobseeker.saved_jobs.includes(:vacancy).order("#{@sort.column} #{@sort.order}")
```

Comment:
Stabbing in the dark but would using the Hash form of `#order` arguments help with your Brakeman woes?

```ruby
.order(@sort.column => @sort.order)
```

Reply:
That's what I would normally do but it doesn't work for association attributes

Reply:
😞  In which case this is fine, given we're explicitly allowing a bunch of attributes.

Reply:
Hang on, I just spotted the Brakeman error. I think what it's _actually_ complaining about is the call to `SavedJobSort#update`, it thinks that `SavedJobSort` is an ActiveRecord model.

In which case you can just tell Brakeman to ignore it: https://brakemanscanner.org/docs/ignoring_false_positives/ (tl;dr: run `brakeman -I`, pick option 2 (Hide previously ignored warnings), and choose `n` (note) as the action, write something poetic, save).